mvc/overview.rst

```diff
@@ -1,8 +1,210 @@
-.. include:: /../common/stub-topic.txt
+Overview of ASP.NET MVC
+=======================
+By `Tom Archer`_, `Daniel Roth`_
 
-|stub-icon| Overview of ASP.NET MVC
-===================================
+.. image:: overview/_static/mvc.png
+  :align: right
 
-.. include:: /../common/stub-notice.txt
+ASP.NET MVC is a lightweight, cross-platform, highly testable Web application development framework that separates an application into three main components: model, view, and controller:
 
-.. _issue: https://github.com/aspnet/Docs/issues/113
+- :doc:`Models </models/index>` are the parts of the application that implement the logic for the application's data domain. Often, model objects retrieve and store model state in a database. For example, a Product object might retrieve information from a database, operate on it, and then write updated information back to a Products table in a SQL Server database.
```

Comment:
This is how ScottGu wrote that in 2007 •"Models" in a MVC based application are the components of the application that are responsible for maintaining state.  Often this state is persisted inside a database (for example: we might have a Product class that is used to represent order data from the Products table inside SQL).
that is state in lieu of data domain.